roles/network_plugin/canal/templates/network.json

```diff
@@ -0,0 +1 @@
+{ "Network": "{{ kube_pods_subnet }}", "SubnetLen": {{ kube_network_node_prefix }}, "Backend": { "Type": "{{ flannel_backend_type }}" } }
```

Comment:
Update: don't need to touch quotes since Flannel expects `SubnetLen` to be a number, not a string and fails otherwise.